.buildkite/scripts/pull-request-pipeline/pull-request-pipeline-steps-8x.yml

```diff
@@ -0,0 +1,71 @@
+steps:
+  # ------------- Unit tests ---------------------
```

Comment:
We don't need to keep `8.x` branch CI steps into `main`, so keeping this file empty would also make sense.